Src/BootCamp.Chapter/Program.cs

```diff
@@ -7,6 +7,24 @@ class Program
         static void Main(string[] args)
```

Comment:
Overall, try to be a tiny bit more subtle with the way you format your code. Separate unrelated pieces of code with a blank line.
 For example reading input [blank line] calculating bmi.
Or reading name [blank line] read age [blank line] ... calculating bmi